visualdl/writer/writer.py

```diff
@@ -612,16 +605,15 @@ def add_roc_curve(self,
             raise RuntimeError("% can't appear in tag!")
         walltime = round(time.time() * 1000) if walltime is None else walltime
         self._get_file_writer().add_record(
-            roc_curve(
-                tag=tag,
-                labels=labels,
-                predictions=predictions,
-                step=step,
-                walltime=walltime,
-                num_thresholds=num_thresholds,
-                weights=weights))
-
-    def add_graph(self, model, input_spec, verbose=False):
+            roc_curve(tag=tag,
+                      labels=labels,
+                      predictions=predictions,
+                      step=step,
+                      walltime=walltime,
+                      num_thresholds=num_thresholds,
+                      weights=weights))
+
+    def add_graph(self, model, input_spec, verbose=False, is_newir=False):
```

Comment:
```suggestion
    def add_graph(self, model, input_spec, verbose=False):
```
add_graph 为用户接口，建议非必要不要新增其他参数。因为后续pir切换上线后，此接口自然就随之切换，不需要额外的接口。